scheduling/shortest_deadline_first.py

```diff
@@ -0,0 +1,70 @@
+"""
+Earliest Deadline First (EDF) Scheduling Algorithm
+
```

Comment:
The code currently sorts processes by deadline each time it needs to find the next process to execute